src/App.js

```diff
@@ -1,25 +1,52 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useState} from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox'
+import AddFoodForm from './components/AddFoodForm'
+import Search from './components/Search'
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
+
+
+
+    const createFood = (newDishes) => {
+      const updatedDishes = [newDishes, ...dishes]
+      setDishes(updatedDishes)
+    }
+    
+    const searchFood = (searchQuery) => {
+    const filteredFood = dishes.filter((food) =>
+      food.name.toLowerCase().includes(searchQuery.toLowerCase())
+      );
+    
+      setDishes(filteredFood);
+    };
+
+    const deleteFood = (name) => {
+      const filteredFood = dishes.filter((dishes) => dishes.name !== name);
+      setDishes(filteredFood);
```

Comment:
Here, since you now have a new `useState` that is the result of the search, you will need to update it as well every time you delete a food. So under the ` setDishes(filteredFood);` add a line like this `setShowFoods(filteredFood)`